phpunit.xml.dist

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Well, maybe the .dist is a way to test some local configs whitout the need of discarding the config file on every commit. This is the point @henriquemoody? Why we will use a different config than the original?